monitoring/prometheus/prometheus.yml

```diff
@@ -6,3 +6,9 @@ scrape_configs:
   - job_name: 'prometheus' 
     static_configs: 
       - targets: ['localhost:9090']
+
+  - job_name: 'django-prometheus'
+    metrics_path: '/prometheus/metrics'
+    static_configs:
+      - targets: ['django:8000']
```

Comment:
@burnerlee why is this still `django:8000`? Can't we pull this from `docker.env`? If not, you'll have to create `prometheus_staging.yaml` and use that.